src/plugins/Provider/view/TableRow.js

```diff
@@ -1,25 +1,48 @@
-const html = require('yo-yo')
+const { h } = require('preact')
+const hyperx = require('hyperx')
+const html = hyperx(h)
 const cuid = require('cuid')
 
 module.exports = (props) => {
-  // const classes = props.active ? 'BrowserTable-row is-active' : 'BrowserTable-row'
   const uniqueId = cuid()
 
+  const checkboxSelect = (ev) => {
+    if (ev.keyCode === 13) {
+      ev.stopPropagation()
+      ev.preventDefault()
+    }
+  }
+
+  const handleItemClick = (ev) => {
+    // when file is clicked, select it, but when folder is clicked, open it
+    if (props.type === 'folder') {
+      return props.handleClick()
+    }
+    props.handleCheckboxClick(ev)
+  }
```

Comment:
nice additions! minor point, should `checkboxSelect` be `handleCheckboxSelect` for consistency? i'm not sure if we follow that convention anywhere else, might be useful to do so though to identify event handlers.

Reply:
in the Transloadit plugin we have some `onXyz` methods, we could address all event listener naming in a separate PR

Reply:
Yeah, good idea. I think it makes sense to call those `handle` because then `onclick=${handleItemClick}` as opposed to `onClick=${onItemClick}`. Sounds more human :-) Maybe.

As for this particular event, calling it `stop` now, since that’s what it does. Unless `handleStop`? :-)